hyperspy/tests/drawing/test_mpl_testing_setup.py

```diff
@@ -16,14 +16,35 @@
 # along with  HyperSpy.  If not, see <http://www.gnu.org/licenses/>.
 
 import matplotlib
-import pytest
 from distutils.version import LooseVersion
+import pytest
+
+from hyperspy.misc.test_utils import check_running_tests_in_CI
 
 
-def test_mlp_agg_for_testing():
-    assert matplotlib.get_backend() == 'agg'
+def test_mlp_agg_for_CI_testing():
```

Comment:
For CI, some tests will work and other will hold the test.
For example, this was the case with [one test in appveyor](https://ci.appveyor.com/project/hyperspy/hyperspy/build/1.0.2106), which was on hold, because it was waiting for a figure to be closed, or something similar! This happen because there was a `plt.show()` call in one function somewhere.
The question is then: where is the bug? Is it the `plt.show()` call, or is it the test?
By having this test, I was thinking, that it would be easier to find out what is wrong.